node/tests/SharedTests.ts

```diff
@@ -1751,6 +1751,44 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `zrank test_%p`,
+        async (protocol) => {
+            await runTest(async (client: BaseClient) => {
+                const key1 = uuidv4();
+                const key2 = uuidv4();
+                const membersScores = { one: 1.5, two: 2, three: 3 };
+                expect(await client.zadd(key1, membersScores)).toEqual(3);
+                expect(await client.zrank(key1, "one")).toEqual(0);
+
+                const version = await getVersion();
+
+                if (version[0] > 7 || (version[0] === 7 && version[1] >= 2)) {
```

Comment:
can you create a util function checkIfServerVersionLessThan (same as check_if_server_version_lt in python), and export it - rather than exporting the getVersion function